From e7bfd27daf00f799a16fccc8d064f166cc8c6661 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?D=2E=20Andrei=20M=C4=83ce=C8=99?=
 <andrei@unlegacy-android.org>
Date: Fri, 13 Apr 2018 15:35:40 +0300
Subject: [PATCH 10/11] qcom_target: Also allow custom HAL paths for non-CAF
 devices
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

For similar reasons as Iee3723cf251d0f485a77a17fd61cb62178833582.

Change-Id: Ibb66b1d11eb1f589edff0490bac71f2ce82c2a94
Signed-off-by: D. Andrei Măceș <andrei@unlegacy-android.org>
---
 build/core/qcom_target.mk | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/build/core/qcom_target.mk b/build/core/qcom_target.mk
index 3e70b013..63203a98 100644
--- a/build/core/qcom_target.mk
+++ b/build/core/qcom_target.mk
@@ -100,15 +100,15 @@ $(call bt-vendor-set-path-variant,bt-caf)
 
 else
 
-$(call project-set-path,qcom-audio,hardware/qcom/audio/default)
-$(call project-set-path,qcom-display,hardware/qcom/display/$(TARGET_BOARD_PLATFORM))
-$(call project-set-path,qcom-media,hardware/qcom/media/$(TARGET_BOARD_PLATFORM))
-
-$(call project-set-path,qcom-camera,hardware/qcom/camera)
-$(call project-set-path,qcom-gps,hardware/qcom/gps)
-$(call project-set-path,qcom-sensors,hardware/qcom/sensors)
-$(call project-set-path,qcom-loc-api,vendor/qcom/opensource/location)
-$(call project-set-path,qcom-dataservices,$(TARGET_DEVICE_DIR)/dataservices)
+$(call set-device-specific-path,AUDIO,audio,hardware/qcom/audio/default)
+$(call set-device-specific-path,DISPLAY,display,hardware/qcom/display/$(TARGET_BOARD_PLATFORM))
+$(call set-device-specific-path,MEDIA,media,hardware/qcom/media/$(TARGET_BOARD_PLATFORM))
+
+$(call set-device-specific-path,CAMERA,camera,hardware/qcom/camera)
+$(call set-device-specific-path,GPS,gps,hardware/qcom/gps)
+$(call set-device-specific-path,SENSORS,sensors,hardware/qcom/sensors)
+$(call set-device-specific-path,LOC_API,loc-api,vendor/qcom/opensource/location)
+$(call set-device-specific-path,DATASERVICES,dataservices,$(TARGET_DEVICE_DIR)/dataservices)
 
 $(call wlan-set-path-variant,wlan)
 $(call bt-vendor-set-path-variant,bt)
-- 
2.17.0

